### DISCUSSION TITLE
Can you tell me what the default screen saver pixel dimensions should be?

### DISCUSSION THREAD

**Discussion by paulyboy** [tanujnotes/Olauncher#388] [Q&A]
Wallpaper seems to scale to wrong size. Seems to zoom and be oversized and not scale to device screen size. What is the pixel dimension that would work?

**Comment by tanujnotes** [comment]
Are you talking about Daily wallpapers of you're applying your own?

**Comment by paulyboy** [comment]
Applying my own wallpaper. Also noticed that it did not bring in any other wallpaper. This is on an android car head unit.

**Reply by tanujnotes** [reply]
You can apply your own wallpaper either from Android settings or Gallery app. Olauncher has no role to play in it.
Maybe you can try switching to default launcher, apply wallpaper and then back to Olauncher.
